openapi/multi-yaml/components/schemas/DisputeResponse.yaml

```diff
@@ -0,0 +1,53 @@
+type: object
+properties:
+  additional_statement:
+    type: string
+    description: any additional evidence or statements
+  cancelation_policy_disclosure:
+    type: string
+    description: an explanation of how and when the customer was shown your refund policy prior to purchase
+  cancellation_rebuttal:
+    type: string
+    description: a justification for why the customer’s subscription was not canceled
+  customer_billing_address:
+    type: string
+    description: the billing address provided by the customer
+  customer_email_address:
+    type: string
+    description: the email address of the customer
+  customer_name:
+    type: string
+    description: the name of the customer
+  customer_purchase_ip_address:
+    type: string
+    description: the IP address that the customer used when making the purchase
+  duplicate_charge_explanation:
+    type: string
+    description: an explanation of the difference between the disputed charge versus the prior charge that appears to be a duplicate
+  product_description:
+    type: string
+    description: a description of the product or service that was sold
+  refund_policy_disclosure:
+    type: string
+    description: documentation demonstrating that the customer was shown your refund policy prior to purchase
+  refund_refusal_explanation:
+    type: string
+    description: justification for why the customer is not entitled to a refund
+  service_date:
+    type: string
+    description: the date on which the customer received or began receiving the purchased service
+  shipping_address:
+    type: string
+    description: the address to which a physical product was shipped
```

Comment:
Do we expect a certain format for this address? If so, could you please add an example?

Reply:
We don't have any specific formats